packages/react-composites/src/composites/CallComposite/pages/UnsupportedBrowser.tsx

```diff
@@ -33,11 +33,20 @@ export const UnsupportedBrowserPage = (props: UnsupportedBrowserPageProps): JSX.
   const unsupportedBrowserStrings = locale.component.strings.UnsupportedBrowser;
   /* @conditional-compile-remove(unsupported-browser) */
   const unsupportedBrowserVersionStrings = locale.component.strings.UnsupportedBrowserVersion;
+  /* @conditional-compile-remove(unsupported-browser) */
+  const unsupportedOperatingSystemStrings = locale.component.strings.UnsupportedOperatngSystem;
 
   /* @conditional-compile-remove(unsupported-browser) */
   let pageElement: JSX.Element | undefined;
   /* @conditional-compile-remove(unsupported-browser) */
-  if (!environmentInfo?.isSupportedBrowser) {
+  if (!environmentInfo?.isSupportedPlatform) {
+    pageElement = (
+      <UnsupportedOperatingSystem
+        onTroubleshootingClick={onTroubleshootingClick}
+        strings={unsupportedOperatingSystemStrings}
+      />
+    );
+  } else if (!environmentInfo?.isSupportedBrowser) {
```

Comment:
what does L63 show:
`return <Stack styles={containerStyles}>{pageElement}</Stack>;`
?

Can we ever hit that line? If not instead do
```
else {
  throw new Error(...
}
```

Reply:
this will return the different unsupportedEvironment UI to the page. I included the error since if the user hits the case that they get to this page without environmentInfo for whatever reason we should throw an error letting them know.